src/lockTracer.cpp

```diff
@@ -74,9 +64,11 @@ void LockTracer::stop() {
 
     // Reset Unsafe.park() trap
     setUnsafeParkEntry(env, _orig_unsafe_park);
+
+    _initialized = false;
```

Comment:
Initialization is supposed to be one-time thing. Calling it multiple times is not only expensive, but is wrong. E.g., you've just introduced another leak where a global JNI reference `_Unsafe` is allocated every time `start` is called.

Reply:
I see, check the latest revision